python/PyQt6/core/conversions.sip

```diff
@@ -1242,6 +1242,24 @@ which are not wrapped by PyQt:
     %End
 };
 
+template <TYPE>
+%MappedType std::unique_ptr< TYPE > /NoRelease,AllowNone,TypeHint="Optional[TYPE]",TypeHintValue="TYPE"/
+{
+%TypeHeaderCode
+#include <memory>
+%End
+
+%ConvertFromTypeCode
+    const sipTypeDef *sip_type = sipFindType("TYPE*");
+    return sipConvertFromNewType(sipCpp->release(), sip_type, NULL);
+%End
+
+%ConvertToTypeCode
+    // only one way for now...
+    return 0;
```

Comment:
I assume if we end up here we'll want an error to be printed somewhere

Reply:
Trying to expose any method which needs python->unique_ptr conversion causes a compilation failure